content/hardware/04.pro/boards/portenta-x8/datasheet/datasheet.md

```diff
@@ -1,571 +1,1161 @@
+
+**IC SAR Warning:**
+
+English
+This equipment should be installed and operated with a minimum distance of 20cm between the radiator and your body.  
+
+French:
+Lors de l’ installation et de l’ exploitation de ce dispositif, la distance entre le radiateur et le corps est d ’au moins 20 cm.
+
+**Important:** The operating temperature of the EUT can’t exceed 85℃ and shouldn’t be lower than -40℃.
+
+Hereby, Arduino S.r.l. declares that this product is in compliance with essential requirements and other relevant provisions of Directive 201453/EU. This product is allowed to be used in all EU member states.
+
+| Frequency bands      | Maximum output power (ERP) |
+|----------------------|----------------------------|
+| 2.4 GHz, 40 channels | +6dBm                      |
+
+## Company Information
+
+| Company name    | Arduino SRL                                   |
+|-----------------|-----------------------------------------------|
+| Company Address | Via Andrea Appiani 25, 20900, MONZA MB, Italy |
+
+## Reference Documentation
+
+| Ref                       | Link                                                                                                |
+|---------------------------|-----------------------------------------------------------------------------------------------------|
+| Arduino IDE (Desktop)     | <https://www.arduino.cc/en/Main/Software>                                                             |
+| Arduino IDE (Cloud)       | <https://create.arduino.cc/editor>                                                                    |
+| Cloud IDE Getting Started | <https://create.arduino.cc/projecthub/Arduino_Genuino/getting-started-with-arduino-web-editor-4b3e4a> |
+| Arduino Pro Website       | <https://www.arduino.cc/pro>                                                                          |
+| Project Hub               | <https://create.arduino.cc/projecthub?by=part&part_id=11332&sort=trending>                            |
+| Library Reference         | <https://github.com/arduino-libraries/>                                                               |
+| Online Store              | <https://store.arduino.cc/>                                                                           |
+
+## Change Log
+
+| **Date**   | **Revision** | **Changes**                            |
+|------------|--------------|----------------------------------------|
+| 17/01/2024 | 5            | Add Temperature and MTBF Tables        |
+| 11/12/2023 | 5            | Add Portenta Hat Carrier compatibility |
+| 07/11/2023 | 4            | Add missing board dimensions           |
+| 26/01/2023 | 3            | Clarify open-source nature of M7 core  |
+| 12/09/2022 | 2            | Make cores clear, minor fixes          |
+| 24/03/2022 | 1            | Release                                |
+=======
+---
+identifier: ABX00049
+title: Arduino® Portenta X8
+type: pro
+author: Ali Jahangiri
+---

```

Comment:
It seems like something happened here @mcmchris @jcarolinares 🤔 

Did the changes include rewriting the full datasheet?

Reply:
Not actually, it just reported me a conflict, and when I was looking at it, to solve it I copied and paste the whole datasheet. I know it's not the best way. I just discussed it with @jcarolinares